theano/tensor/nnet/tests/test_sigm.py

```diff
@@ -328,14 +316,14 @@ def test_local_ultra_fast_sigmoid(self):
 
```

Comment:
There is a check on l.309 that should be updated or removed, depending on what actually is in the graph, or what optimizations actually get applied.